frontend/packages/console-app/src/redux/reducers/guided-tour-reducer.ts

```diff
@@ -0,0 +1,40 @@
+import { GuidedTourSidebarActions, Actions } from '../actions/guided-tour-actions';
+import { RootState } from '@console/internal/redux';
+import { GuidedTourStatus } from '../../components/guided-tours/utils/guided-tour-status';
+
+export const guidedTourReducerName = 'guidedTour';
+
+type State = {
+  isExpanded: boolean;
+  activeTourID: string;
+};
+
+const initialState: State = {
+  isExpanded: false,
+  activeTourID: '',
+};
+
+export default (state = initialState, action: GuidedTourSidebarActions) => {
+  if (action.type === Actions.ToggleGuidedTourSidebarExpanded) {
+    return {
+      isExpanded:
+        state.isExpanded &&
+        action.payload.activeTourID &&
+        action.payload.activeTourID !== state.activeTourID
+          ? state.isExpanded
+          : !state.isExpanded,
+      activeTourID: action.payload.activeTourID,
+    };
+  }
+
+  return state;
+};
+
+export const isGuidedTourSidebarExpanded = (state: RootState): boolean =>
+  !!state.plugins?.console?.[guidedTourReducerName]?.isExpanded;
+
+export const getActiveTourID = (state: RootState): string =>
+  state.plugins?.console?.[guidedTourReducerName]?.activeTourID;
+
+export const getActiveTourStatus = (state: RootState): GuidedTourStatus =>
+  state.plugins?.console?.[guidedTourReducerName]?.activeTourStatus;
```

Comment:
Remove.
Beyond the scope of this story.
```suggestion
```

Reply:
done